sdk/src/main/java/com/hcaptcha/sdk/HCaptchaConfig.java

```diff
@@ -119,4 +120,8 @@ public class HCaptchaConfig implements Serializable {
      */
     @Builder.Default
     private long tokenExpiration = 120;
+
+    @JsonIgnore
+    @Builder.Default
+    private IHCaptchaHtmlProvider htmlProvider = new HCaptchaHtml();
```

Comment:
Should be package private and not "settable" via the builder.

Reply:
Done